pkg/config/helm_values_test.go

```diff
@@ -14,6 +15,8 @@ func TestEnvironmentExposecontrollerHelmValues(t *testing.T) {
 
 	testFile, err := ioutil.ReadFile("helm_values_test.yaml")
 	assert.NoError(t, err)
+	helmValuesFromFile := config.HelmValuesConfig{}
+	err = yaml.Unmarshal(testFile, &helmValuesFromFile)
```

Comment:
`assert.NoError(t, err)`?

Reply:
again, if the unmarshall fails then the assertion will also fail, but no hard in putting it in.